packages/core/src/controllers/OptionsController.ts

```diff
@@ -176,23 +188,33 @@ export const OptionsController = {
     state.metadata = metadata
   },
 
-  setOnrampEnabled(enableOnramp: OptionsControllerState['enableOnramp']) {
+  setOnrampEnabled(
+    enableOnramp: OptionsControllerState['enableOnramp'] = ConstantsUtil.DEFAULT_FEATURES.onramp
```

Comment:
When you already pass the default value, do you need the type definition?
```suggestion
    enableOnramp = ConstantsUtil.DEFAULT_FEATURES.onramp
```

Reply:
We need it because the param has multiple types and the constant just is one of them.

Reply:
We could force type the constant, but it is going to be circular dependency